### ISSUE TITLE
Better accessibility labels for button bar for contacts and groups

### ISSUE THREAD

**Issue by MarcoZehe** [threema-ch/threema-web#654]
## Expected Behavior

The buttons to show the QR code, edit a contact or group, or open the chat for that contact or group all have a distinct label and can be distinguished by screen reader users.

## Current Behavior

The buttons to show the QR code, edit a contact or group, or open the chat for that contact or group all have a label of "chat", and thus cannot be distinguished by screen reader users.

## Possible Solution

Adjust the labels for the buttons to what they really do.

## Steps to Reproduce (for bugs)

1. Open Threema Web with a screen reader running.
2. Select the Contacts tab and then a contact.
3. Examine the buttons below the heading for the contact.
4. Can you guess which button does what? ;)

## Your Environment

* Threema Web version: 2.1.0
* Threema App version: 4.0beta
* Browser name and version: Firefox 63 release.
* Computer operating system and version: Windows 10 Version 1809
* Smartphone operating system and version: iOS 12.0.1

**Comment by ovalseven8** [comment]
@dbrgn 

I get a lot of warning in the Firefox console when using Threema Web:
```
ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 
<input id="input_1" class="ng-pristine ng-untouched…d md-input ng-not-empty" type="password" ng-model="ctrl.password" ng-disabled="ctrl.formLocked" autofocus="" aria-labelledby="aria-label-password-reconnect" translate-attr="{'placeholder': 'welcome.PASSWORD'}" autocomplete="current-password" disabled="disabled" aria-invalid="false" placeholder="Passwort">
```

**Comment by dbrgn** [comment]
@ovalseven8 I believe that is a bug in Firefox, since we are already setting the `aria-labelledby` attribute.
